rednotebook/util/markup.py

```diff
@@ -146,7 +145,11 @@ def get_markup_for_day(day, with_text=True, with_tags=True, categories=None, dat
 
     # Add date if it is not None and not the empty string
     if date:
-        export_string += '= %s =\n\n' % date
+        # Following anchor placeholder will be used as a target for every entry reference
+        # mentioning this entry's date. In postprocessing it will be replaced by relevant HTML tag.
+        date_anchor = 'DATE_ANCHOR_PLACEHOLDER_%s' % day.date.strftime('%Y-%m-%d')
+
+        export_string += '= %s %s =\n\n' % (date_anchor, date)
```

Comment:
I think we can directly put in the HTML code (if the target is '(x)html') by using the txt2tags inline markup, i.e., something like `''<span> ... </span>''`.

Reply:
Unfortunately this doesn't work for title contents. But with this syntax, when the anchor is placed just above the title we don't need so much code in `_get_config` function.
Which is good - thanks :)